TESTS/netsocket/socket_sigio/main.cpp

```diff
@@ -122,10 +126,13 @@ void test_socket_attach() {
         sock.send(buffer, strlen(buffer));
         // wait for recv data
         recvd.wait();
+
+        result = true;
```

Comment:
> as test was always OK even with no connection!

That should not be the case -- [this line](https://github.com/jeromecoutant/mbed/blob/ccd179768ccf61d9ab50e765d82c118df1004efa/TESTS/netsocket/socket_sigio/main.cpp#L124) will attach the `get_data` function. If `get_data` never gets called because of an incorrect implementation of sigio, this test case will time out. If `get_data` does get called, the success of the test depends on the getting some data from the server [here](https://github.com/jeromecoutant/mbed/blob/ccd179768ccf61d9ab50e765d82c118df1004efa/TESTS/netsocket/socket_sigio/main.cpp#L86)

Reply:
No...
With my local test env, test was always OK whereas it is not possible to connect ARM server....